### PR TITLE
get template

### PR DESCRIPTION
**Pull request by victorbalan**

### CODE REVIEW

File: platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/Template.java [hunk 1]
```diff
@@ -40,6 +40,6 @@ public void setFileName(String fileName) {
 
     @Override
     public String filePath() {
-        return String.format("/%s/%s", language, fileName);
+        return String.format("%s", fileName);
```
Comment (flowlo) [inline platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/Template.java]: `String.format("%s", s)` is `s`, therefore `filePath()` will always return `fileName`, so it is obsolete, and we should consider renaming `fileName` to `filePath`.

Comment (flowlo) [inline platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/Template.java]: Also, `filePath` is not used in any of the other changes. Is this a related change?

Comment (victorbalan) [inline platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/Template.java]: this will be changed when we add folder and fileName to the template entity
